### ISSUE TITLE
Integrar con Markdown

### ISSUE THREAD

**Issue by acciobugs** [Gochip/labsis_seg#26]
Estaría bueno poder darle formato con Markdown, sobre todo para cuando queremos resaltar código

**Comment by acciobugs** [comment]
averigüé y para CKeditor viene un plugin, Code Snippet, y otro más, Code Snippet GHeShi. La versión de CKeditor se podría actualizar? o no es necesario por ahora?

**Comment by Gochip** [comment]
Sí. Pienso que deberíamos actualizar a la 4. Creo que la 5 es solo compatible con Angular pero no estoy seguro.

Si alguien lo averigua, que avise.

**Comment by acciobugs** [comment]
en la branch de testing actualicé a la 4.12, lo que hay que ver es que al renderizar el artículo quede el código en el formato que se visualiza al editar (con el plugin que ya viene) . queda todo en la misma fuente luego de guardarlo...
